documentation/production/tutorial-production-mode-basic.asciidoc

```diff
@@ -84,3 +84,102 @@ When minifying code, it is often obscured, which makes it harder to read, hence
 Bundling is an optimization where we merge multiple files to a single collection so that the browser doesn't need to
 request so many files from the server. 
 This makes the application load faster.
+
+== Checklist for the production mode app artifact
```

Comment:
Not sure if this should be in this chapter, or its own with the high-level title of something like "Details for the production build" or whatnot

Reply:
Yeah I agree: this page deals with production mode basics; sounds a bit of a stretch to discuss the particularities of app packaging straight in the intro article. If you agree, I'll create a new article instead.

Reply:
Please do so